### PR TITLE
Scenario 07: refactoring

### PR DESCRIPTION
**Pull request by capt-t**
Mainly refactoring of `scenario_07`.

The scenario with improved code is the the same with a few hardly noticable changes:

Enhancements:

- Fix spelling.
- Give reasons for victory/defeat.

Modifications:

- Reorder sentences in objective.

Following Pull Requests:

- The central nebula is now centered around the center, not the ship (which is close to the center anyway).

Now it is easy to move the entire area to a different center or even create several of those. Moreover, is is also easier to reuse parts of the code.

### CODE REVIEW

**Comment by capt-t** [comment]
Wait with merging, there will be an updated version of these commits.

**Comment by capt-t** [comment]
Should be ready for merge.

**Comment by capt-t** [comment]
Should be ready for merge.

**Comment by capt-t** [comment]
Should be ready for merge.

File: scripts/scenario_07_quick_basic.lua [hunk 1]
```diff
@@ -347,14 +352,18 @@ function update(delta)
         -- Note that players can win even if they destroy the enemies by blowing themselves up.
         if enemy_count == 0 then
             victory("Human Navy")
-            setBanner("Mission: SUCCESS")
+            local text = string.format("Mission: SUCCESS (%d seconds left)", math.floor(gametimeleft))
```
Comment (daid) [inline scripts/scenario_07_quick_basic.lua:355]: Does this text fit in the global message box? I remember the font being quite large and the box not being super large.

Comment (capt-t) [reply scripts/scenario_07_quick_basic.lua:355]: Probably not, but this should not be a problem. Compare the detailed mission result in Borderline Fever.

Is there another possibility for a longer interesting message?

If it is a problem, we can remove this detail. The other changes (and the upcoming) are more relevant.

**Pull request merged**